### PR TITLE
参考#26 修复媒体库问题

### PR DESCRIPTION
**Pull request by sgxnew**
参考#26 修复媒体库些许问题：
1、修复电视直播异常问题；
2、修复可能进入媒体库异常问题；
……

### CODE REVIEW

**Comment by jackloves111** [comment]
image
经测试，进入媒体库异常问题和#26 一样，是有效的。但是手机端还是不行。

**Comment by jackloves111** [comment]
fbd66de0-a6f6-4831-99bd-7dcf28fb6b3a
找到原因了。模拟平板是可以的，说明不是手机或者电脑设备的问题，是当屏幕宽度低于560px时，会取消轮播图显示。只有媒体库显示，这个时候刷新首页 ，点击媒体库会不显示。当屏幕宽度大于560px时，正常。

可以加个判断当屏幕小于560px，直接恢复到原版的样式。这样应该就可以了

**Comment by sgxnew** [comment]
是的，项目本身就没有适配手机；$(".view:not(.hide) .section0 .emby-scroller .itemsContainer")[0].items属性值.detach().appendTo()移动位置后没有带过去，重新赋值，在PC端上可以正常，在手机上重新赋值后不知怎么又被清空。

**Comment by sgxnew** [comment]
> image 经测试，和#26 一样，电脑网页是有效的。但是手机端网页还是不行。不过影响不大，手机端一般用客户端。

调了一下位置应该可以了，试一下

**Comment by sgxnew** [comment]
image

**Comment by jackloves111** [comment]
经测试，现在确实可以了。感谢大佬的PR。这样这个版本就趋近完美了！

**Comment by jackloves111** [comment]
@Nolovenodie 大佬大佬。可以关闭#26。然后把这个PR同步了

**Comment by jackloves111** [comment]
大佬怎么把这个关掉了
